examples/05_stable_diffusion/scripts/download_pipeline.py

```diff
@@ -25,6 +26,22 @@
     help="pipeline files local directory",
 )
 def download_pipeline_files(token, save_directory) -> None:
+    
+    """
+        Set the OS environment variable AITEMPLATE_WORK_DIR to point to an absolute path to a directory which 
+        will be used to save the AIT compiled model artifacts. Make sure the OS user running this script has read and write 
+        permissions to this directory. By default, the artifacts will be saved under tmp/ folder of the 
+        current working directory. 
+    """
+
+    env_name = "AITEMPLATE_WORK_DIR"
+    try:
+        if os.environ[env_name]:
+            save_directory = os.path.join(os.environ[env_name], 'diffusers-pipeline', 'stabilityai','stable-diffusion-v2')
+            print("The value of", env_name, " is ", save_directory)
+    except KeyError:
+        print("AITEMPLATE_WORK_DIR environment variable is not set. Using default local dir as ",save_directory)
+
```

Comment:
Ditto about 1) indentation, 2) usage of try/except, 3) spaces between args, 4) string formatting.